docs/angular/guides/setup-incremental-builds.md

```diff
@@ -81,7 +87,7 @@ Change your Angular app’s executor to @nrwl/angular:webpack-browser and the 
 
 ## Running and serving incremental builds
 
-To build an app incrementally use the following commands.
+To build an app incrementally use the following command:
 
 ```bash
 nx build myapp --with-deps --parallel
```

Comment:
Is `--with-deps` still needed? I remember we wanted to replace it with some config level settings instead. Maybe that hasn't been merged/released yet though

Reply:
Good catch!

I think it's not needed if the project is configured with `dependsOn` or if there's a global `targetDependencies` in nx.json. This repo is already using `targetDependencies`, so, it's definitely available, but I don't see any of those properties in nx.dev yet or any deprecation notice for `with-deps`.

Reply:
I can still see `--with-deps` even in the tutorial https://nx.dev/latest/angular/tutorial/10-computation-caching#--with-deps, so I guess there has to be a full review later of that property usage to update the docs. What do you think?

Reply:
Yep, let's do that